CHANGELOG.md

```diff
@@ -1,9 +1,126 @@
-# dbt_sap version.version
+# dbt_sap v0.2.0
+[PR #25](https://github.com/fivetran/dbt_sap/pull/25) includes the following updates:
 
-## Documentation
-- Added Quickstart model counts to README. ([#10](https://github.com/fivetran/dbt_sap/pull/10))
-- Corrected references to connectors and connections in the README. ([#10](https://github.com/fivetran/dbt_sap/pull/10))
+## Schema & Data Updates
+**71 total changes • 0 possible breaking changes.**
```

Comment:
just realized the new column additions to `stg_sap__t001` are schema/reaking changes, as are the new column additions to `stg_sap__lfa1`. So I'd add those as rows, and this is now

```suggestion
**73 total changes • 2 possible breaking changes.**
```

Reply:
You also might have to add the new columns to the "New Name" column for these two models, even though it is a lot of new columns...